addons/koji/common/src/main/java/org/commonjava/indy/koji/content/KojiContentManagerDecorator.java

```diff
@@ -410,7 +412,7 @@ private Transfer transferBuildArtifact( final KojiArchiveInfo archive, final Rem
         {
             try
             {
-                Transfer target = delegate.getTransfer( hosted, path, TransferOperation.DOWNLOAD );
+                ValueHolder<Transfer> target = new ValueHolder<>();
```

Comment:
I've started replacing these with AtomicReference<>. I'm not sure if it's correct, but this was only ever an ad hoc solution...it'd be nice to use something standard. Not critical right now, but something to keep in mind.